cscs-checks/libraries/math/scalapack_compile_run.py

```diff
@@ -27,12 +27,14 @@ def __init__(self, linkage):
         self.maintainers = ['CB', 'LM', 'MKr']
         self.tags = {'production'}
 
-    def setup(self, environ, partition, **job_opts):
-        super().setup(environ, partition, **job_opts)
+    def setup(self, partition, environ, **job_opts):
         if (self.current_system.name in ['kesch', 'monch'] and
-            self.current_environ.name == 'PrgEnv-gnu'):
+            environ.name == 'PrgEnv-gnu'):
```

Comment:
We could remove Monch, no?

Reply:
My take is yes, cos @victorusu will provide his own tests at some time in the future.

Reply:
One Question could be, whether this should be also done in this PR for the Kesch tests only or in a separate issue going through all the tests at once?

Reply:
We have opened issue #489 to address this one.